### DISCUSSION TITLE
skip the remaining phases of current plugins and the rest of plugins

### DISCUSSION THREAD

**Discussion by zffocussss** [Kong/kong#8042] [Help] [answered]
Hi everyone:
   Is there a way to skip the remaining phases of current plugins and the rest of the plugins (especially global plugins) when some conditions are matched?
  I am using kong as an HTTP gateway to serve multi ingress based on multi-domains. such as a.domain.com and b.domain.com.
  I want to make requests matching **a.domain.com** go through the normal plugin logics(access, body, response..etc).in the other hand, make requests skip matching **b.domain.com** all the plugins logics, ie kong just proxy the requests to upstream transparently, without any kong plugins involved in.

**Comment by Godspeed2021** [comment]
[]()

**Comment by kikito** [comment, marked as answer]
Create one Route (or service) for traffic going to `a.domain.com` and a different one for `b.domain.com`. Then apply plugins to the first one and not the second. And don't use global plugins if you don't need them in absolutely everything.

**Reply by zffocussss** [reply]
you are right
